aries-site/src/layouts/navigation/NavPage.js

```diff
@@ -16,21 +16,26 @@ const NavItem = ({ item, topic }) => {
   return (
     // Need to pass href because of: https://github.com/zeit/next.js/#forcing-the-link-to-expose-href-to-its-child
     <Link href={`/${topic}/${formattedItem}`} passHref>
-      <Box
-        direction="row"
-        margin={{ vertical: 'large' }}
-        gap={size !== 'small' ? 'large' : 'medium'}
-      >
-        {/* Adds placeholder icon for the meantime
-         * while we wait to get all the icons */}
-        {itemData.icon ? itemData.icon('xlarge') : iconsMap.branding('xlarge')}
-        <Box>
-          <Text weight="bold" size={size !== 'small' ? 'large' : 'medium'}>
-            {itemData.name}
-          </Text>
-          <Paragraph size="small">{itemData.description}</Paragraph>
+      {/* Needs to be <a> in DOM for web crawling: https://support.google.com/webmasters/answer/9112205?hl=en */}
+      <Button fill>
```

Comment:
This approach makes sense to me.

Reply:
I believe the `passHref` prop from line 18 is getting passed to the button, which then is being treated as `<Button>` with an `href` prop.

Reply:
Yes, what Matt said is correct, the passHref puts the href on the button and then Grommet knows to make it an <a> in the DOM.

Reply:
Correct, I was looking for the `href` on the Button level.  thanks for clarifying.